src/busy-indicator.scss

```
@@ -51,6 +52,29 @@ $fd-dots-size-l: 2rem;
   background-color: currentColor;
 }
 
+.#{$block}-extended {
+  @include fd-flex-center() {
+    flex-direction: column;
+  };
+
+  &.#{$block}-extended--message-toast {
```

Do we need the entire prefix `&.#{$block}-extended`?  How about just `&--message-toast`: 
```suggestion
  &--message-toast {
```

---

`&--message-toast` won't enforce the specificity.
we want to explicitly say that `fd-busy-indicator-extended` is used at the same level with `fd-busy-indicator-extended--message-toast`  aka we have `class="fd-busy-indicator-extended fd-busy-indicator-extended--message-toast"`.
This way, because 2 classes used together on a same level is stronger that `fd-message-toast` and that's why they overwrite its rules.

---

ah I get it now. Cool. Thanks for explaining!